eksconfig/add-on-node-groups.go

```diff
@@ -232,6 +243,10 @@ func (cfg *Config) validateAddOnNodeGroups() error {
 			return fmt.Errorf("%q both ImageID and ImageIDSSMParameter are empty", cur.Name)
 		}
 
+		if cur.ClusterAutoScaler.Enable && !cfg.AddOnNodeGroups.RoleCreate {
```

Comment:
Also error if `CA.Enable == true` and `AddOnMetricsServer.Enable == false`? Is metrics-server requirement for CA?

Reply:
> Also error if `CA.Enable == true` and `AddOnMetricsServer.Enable == false`? Is metrics-server requirement for CA?

metrics-server is not a requirement for CA. Metrics server is a requirement for HPA.

Reply:
ack